ui/app/components/task-sub-row.hbs

```diff
@@ -0,0 +1,79 @@
+<tr class="task-sub-row"
+	{{keyboard-shortcut
+		enumerated=true
+		action=(action "gotoTask" @allocation this.task)
+	}}
+>
+	<td colspan={{@namespan}}>
+		/
+		<LinkTo @route="allocations.allocation.task" @models={{array @allocation this.task}}>
+			{{this.task.name}}
+		</LinkTo>
+		{{!-- TODO: in-page logs --}}
+		{{!-- <FlightIcon @name="logs" /> --}}
```

Comment:
(very temporary) placeholder: covered in https://github.com/hashicorp/nomad/issues/14579